packages/communication-react/rollup.config.js

```diff
@@ -3,15 +3,27 @@
 
 import Package from './package.json';
 import commonConfig from '../../common/config/rollup/rollup.config';
+import commonjs from '@rollup/plugin-commonjs';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import svg from 'rollup-plugin-svg';
+import json from '@rollup/plugin-json';
 
 export default [
   {
     ...commonConfig(Package),
     input: './dist/dist-esm/communication-react/src/index.js',
     output: {
-      file: './dist/dist-cjs/communication-react/index.js',
+      dir: './dist/dist-cjs/communication-react',
```

Comment:
what does changing from file to dir do here?

Reply:
Not why I was getting error `[!] Error: You must set either output.file for a single-file build or output.dir when generating multiple chunks.` at first but reverted it back